near/contracts/bridge/src/lib.rs

```diff
@@ -375,6 +391,75 @@ impl FastBridge {
         U128::from(0)
     }
 
+    #[pause(except(roles(Role::UnrestrictedUnlock)))]
+    pub fn unlock_and_withdraw(
```

Comment:
The comments should be added for new functions

Reply:
166d191c08dd81100ef9f5296dabc07565693757